apps/examples/power/power_main.c

```diff
@@ -33,54 +33,51 @@
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
-#define PM_LOCK_PATH						"/proc/power/domains/0/pm_lock"
-#define PM_UNLOCK_PATH					"/proc/power/domains/0/pm_unlock"
-#define PM_SLEEP_PATH						"/proc/power/domains/0/enter_sleep"
+#define POWER_THREAD_SIZE				2048
+#define POWER_THREAD_PRIORITY				255
+#define PM_DAEMON_INTERVAL				5
 
-#define POWER_THEAD_SIZE				2048
-#define POWER_THEAD_PRIORITY		255
-#define PM_DAEMON_INTERVAL			15
-#define PM_LOCK(x)								power_set_int(PM_LOCK_PATH, x)
-#define PM_UNLOCK(x)							power_set_int(PM_UNLOCK_PATH, x)
+#define PM_WAKEUP_TIMER_DURATION			5
```

Comment:
Did you add the units in comment already? It is not existing here, also the timer duration should be in microseconds precision, you are setting sec as precision here, please revise accordingly/

Reply:
Push failed, sorry I did not notice.
Thankyou for pointing out